syntax = "proto3";

package messagedef;

service SampleData {

    rpc SendData (stream SampleDataMessage) returns (Response) {}
}

message SampleDataMessage {
    string stringField = 1;
    int32 numberField = 2;
    bytes fileFieldChunk = 3;
}

message Response {
    string message = 1;
}
